src/content/plugins/html-webpack-plugin.md

```diff
@@ -51,12 +52,9 @@ This will generate a file `dist/index.html` containing the following:
 </html>
 ```
 
-If you have multiple webpack entry points, they will all be included with `script`
-tags in the generated HTML.
+If you have multiple webpack entry points, they will all be included with `<script>` tags in the generated HTML.
```

Comment:
**in** the generated HTML -> **inside** the generated HTML

Reply:
I would keep **in**, why change?

Reply:
Hmm... Ok